### PR TITLE
feat(userCreate/B01) add controllers userCreate and Route

### PR DESCRIPTION
**Pull request by FranzDuran**
cree una ruta post para usuario 
userRoutes/Routes
[userCreate/controllers](url)
   ---------------------------------------
Evidencia de que se creo el usuario correctamente
image

En caso de ERROR se enviara este mensaje en el postman
image

### CODE REVIEW

**Review by iSensative — APPROVED** [review]
All ok

File: api/controllers/UserControllers/UserCreate.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          throw new ErrorObject("That email is already in use", 400)
+      }
+      
+      const response = await User.create({
+        fullname, email, password, phone
+      })
+      endpointResponse({
+        res,
+        code:201,
+        message: 'Test retrieved successfully',
```
Comment (miguelbel00) [inline api/controllers/UserControllers/UserCreate.js:26]: change the message for the apropiated

**Review by miguelbel00 — CHANGES_REQUESTED** [review]
Peding evidence that the endpoint is working

File: api/controllers/UserControllers/UserCreate.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const {ErrorObject} = require("../../helpers/error")
+const createHttpError = require('http-errors')
+const { User } = require('../database/models')
+const { endpointResponse } = require('../helpers/success')
```
Comment (miguelbel00) [inline api/controllers/UserControllers/UserCreate.js:4]: This imports are working?

**Review by miguelbel00 — CHANGES_REQUESTED** [review]
Check when finish the trello card that the app is keep working
## ScreenShot
imagen

**Review by miguelbel00 — APPROVED** [review]
All Ok

**Pull request merged**